scripts/Muon/GUI/Common/plot_widget/data_pane/plot_group_pair_model.py

```diff
@@ -0,0 +1,55 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from Muon.GUI.Common.utilities.run_string_utils import run_list_to_string
+from Muon.GUI.Common.plot_widget.base_pane.base_pane_model import BasePaneModel
+from Muon.GUI.Common.ADSHandler.workspace_naming import *
+
+
+class PlotGroupPairModel(BasePaneModel):
+
+    def __init__(self, context, name):
+        super().__init__(context, name)
+
+    def create_tiled_keys(self, tiled_by):
+        if tiled_by == "Group/Pair":
+            keys = self.context.group_pair_context.selected_groups_and_pairs
+        else:
+            keys = [run_list_to_string(item) for item in self.context.data_context.current_runs]
+        return keys
+
+    def _create_workspace_label(self, workspace_name, index):
+        group = str(get_group_or_pair_from_name(workspace_name))
+        run = str(get_run_numbers_as_string_from_workspace_name(workspace_name, self.context.data_context.instrument))
+        instrument = self.context.data_context.instrument
+        rebin_label = self._get_rebin_label(workspace_name)
+        if not self.context.plot_panes_context[self.name]._is_tiled:
+            return "".join([instrument, run, ';', group, rebin_label])
```

Comment:
Using f-strings will make this easier to read

Reply:
would it be
` f"{instrument}{run}; {group}{rebin_label}"

Reply:
Yes, but without the space after the semi-colon if we want it the same as above

Reply:
I think using f-string for the other places where we have join would also be a good idea